src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/CheckLatestConsentOrderController.java

```diff
@@ -45,11 +46,10 @@
         Map<String, Object> caseData = callback.getCaseDetails().getData();
         long caseId = callback.getCaseDetails().getId();
 
-        if (isNull(caseData.get("latestConsentOrder"))) {
+        if (isNull(caseData.get(LATEST_CONSENT_ORDER))) {
             log.info("Failed validation for {} as latest Consent Order field was null", caseId);
             return ResponseEntity.ok(AboutToStartOrSubmitCallbackResponse.builder()
-                .errors(Arrays.asList("Latest Consent Order Field is empty. "
-                        + "Please use the Upload Consent Order Event instead of Send Order"))
+                .errors(Arrays.asList("Latest Consent Order Field is empty. Please use the Upload Consent Order Event instead of Send Order"))
```

Comment:
Would it be a good idea to store error messages in a dedicated file and call the consts for them ?